### PR TITLE
Minor changes to FAQ.md

### PR DESCRIPTION
**Pull request by Buedenbender**
Minor improvements to the phasingout de2. 
Additionally, the .md linter/autoformatter might made some minor
adjustments to formatting.

### CODE REVIEW

File: FAQ.md [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
 
 ## Table of Contents
+
 - [Why should I use this service?](#why-should-i-use-this-service)
 - [Why choose Tunnel⚡Sats over other VPN providers?](#why-choose-tunnelsats-over-other-vpn-providers)
 - [Is your service reliable?](#is-your-service-reliable)
 - [What services are used?](#what-services-are-used)
 - [What about data storage and privacy?](#what-about-data-storage-and-privacy)
-- [Do you store my data? If so, which one and how do you use it?](#do-you-store-my-data-if-so-which-one-and-how-do-you-use-it)
+- [Do you store my data? If so, which one and how do you use it?](#do-you-store-my-data-if-so-which-oneP-and-how-do-you-use-it)
```
Comment (blckbx) [inline FAQ.md:10]: This doesn't look right, why the change?

File: FAQ.md [hunk 2]
```diff
@@ -238,40 +273,61 @@ table inet excludeTraffic {
...
-- restart your lightning implementation
+In fact there are five simple steps to take:
+
+1. Edit your `tunnelsatsv2.conf` and change `Endpoint` entry to `Endpoint = de3.tunnelsats.com:51820`
+2. Fetch latest version of setup script: `wget -O setupv2.sh https://github.com/blckbx/tunnelsats/raw/main/scripts/setupv2.sh`
+3. Run it: `sudo bash setupv2.sh`
+4. Edit your lightning config file and change the DNS entry accordingly:
+   - LND: `externalhosts=de3.tunnelsats.com:<yourVPNport>`
+   - CLN: `announce-addr=de3.tunnelsats.com:<yourVPNport>`
+   - As allways before changing the config, it is good practice to backup your config.
```
Comment (blckbx) [inline FAQ.md:299]:
```suggestion
   - As always before changing the config, it is good practice to backup your config.
```

File: FAQ.md [hunk 3]
```diff
@@ -238,40 +273,61 @@ table inet excludeTraffic {
...
+
+5. Restart your lightning implementation
+
+**Verify the switch to de3.tunnelsats.com**  
+To verify that the switch worked, you can run `lncli getinfo | jq '.uris'`. The public IP should start with 159.x.x.x 
+
+**Where do I find the config files?**  
+If you can not locate a file (e.g., the `tunnelsatsv2.conf` or your lightning config) you can allways use the `find` command.
+
+- Locate tunnelsatsv2.conf: `sudo find / -name "tunnelsatsv2.conf"`
```
Comment (blckbx) [inline FAQ.md:311]: just a note: there're 2 tunnelsatsv2.conf files. one that the user downloaded, the other one got extended with firewall rules and is located in /etc/wireguard/

File: FAQ.md [hunk 4]
```diff
@@ -238,40 +273,61 @@ table inet excludeTraffic {
...
+- Locate lnd.conf: `sudo find / -name "lnd.conf"`
+- Locate cln.conf: `sudo find / -name "cln.conf"`
```
Comment (blckbx) [inline FAQ.md:313]:
```suggestion
- Locate cln.conf: `sudo find / -name "config"`
```
This is a tough one. CLN's config file is just named `config` (see https://lightning.readthedocs.io/lightningd-config.5.html#synopsis). If you know a way to explicitly search for a file named `config` (without whitespaces) that'd be good.

**Review by blckbx — COMMENTED** [review]
Thanks for your contribution! 
A few nits I got.

File: FAQ.md (hunk 1 above)
Comment (Buedenbender) [reply FAQ.md:10]: Good catch, I think this slipped in accidently

File: FAQ.md (hunk 3 above)
Comment (Buedenbender) [reply FAQ.md:311]: Should we add it to the readme? So that it is not confusing to users?

File: FAQ.md (hunk 4 above)
Comment (Buedenbender) [reply FAQ.md:313]: I am on it finding a better way to locate it. In the docs I found that it is usually stored in one of three places, so this could be a workaround.

File: FAQ.md (hunk 2 above)
Comment (Buedenbender) [reply FAQ.md]: Done 😊

File: FAQ.md (hunk 3 above)
Comment (blckbx) [reply FAQ.md:311]: Yes, probably a good idea.

File: FAQ.md (hunk 4 above)
Comment (blckbx) [reply FAQ.md:313]: https://github.com/blckbx/tunnelsats/blob/main/FAQ.md#where-do-i-find-my-lightning-configuration-file

File: FAQ.md (hunk 3 above)
Comment (Buedenbender) [reply FAQ.md:316]: What do you think?

File: FAQ.md (hunk 4 above)
Comment (Buedenbender) [reply FAQ.md]: Currently I linked the heading, still figuring out a command line way to locate. Actually the cli for CLN is I believe called [lightning-cli](https://github.com/ElementsProject/lightning#configuration-file), but I think there is no method implemented for locating the config. I tried reaching out in IRC in the meanwhile

File: FAQ.md [hunk 5]
```diff
@@ -238,40 +273,74 @@ table inet excludeTraffic {
...
+     - To backup the conf use `sudo cp <path/to/conf> <path/where/tosafe/backup>`
+     - E.g., for raspiblitz and lnd: `sudo cp /mnt/hdd/lnd/lnd.conf ~/lnd_backup.conf`
+
+5. Restart your lightning implementation
+
+**Verify the switch to de3.tunnelsats.com**  
+To verify that the switch worked, you can run
+
+- LND: `lncli getinfo | jq '.uris'`
+- CLN: _tbd_
```
Comment (Buedenbender) [inline FAQ.md:309]: Still have to verify but one possible options is[`lightning-cli getinfo`](https://github.com/ElementsProject/lightning/pull/5244) 
> JSON-RPC: Adds dynamically detected public IP addresses to getinfo (#5244)

**Review by Buedenbender — COMMENTED** [review]
Needs verification: found a way to get public IP in CLN

File: FAQ.md (hunk 3 above)
Comment (blckbx) [reply FAQ.md:316]:
```suggestion
- Locate tunnelsatsv2.conf: `sudo find / -maxdepth 5 \( -path /etc/wireguard -prune -o -path /sd-root -prune \) -o -type f -name "tunnelsatsv2.conf"`
```
omits `/etc/wireguard` (used by wireguard) to find the originally downloaded file

File: FAQ.md (hunk 5 above)
Comment (blckbx) [reply FAQ.md:309]:
```suggestion
- CLN: `lightning-cli getinfo | jq '.address'`
```

File: FAQ.md (hunk 3 above)
Comment (Buedenbender) [reply FAQ.md:316]: Tested it and it works somewhat (atleast omits file name from /etc/wireguard)
> /etc/wireguard
> /home/admin/tunnelsats/tunnelsatsv2.conf

**Comment by Buedenbender** [comment]
sorry took me a while but incorporated both suggestions :)

File: FAQ.md [hunk 6]
```diff
@@ -238,40 +273,74 @@ table inet excludeTraffic {
...
+
+- LND: `lncli getinfo | jq '.uris'`
+- CLN: `lightning-cli getinfo | jq '.address'`
+
+The public IP should start with 159.x.x.x
+
+**Where do I find the config files?**  
+If you can not locate a file (e.g., the `tunnelsatsv2.conf` or your lightning config) you can always use the `find` command.
+
+- Locate tunnelsatsv2.conf: `sudo find / -maxdepth 5 \( -path /etc/wireguard -prune -o -path /sd-root -prune \) -o -type f -name "tunnelsatsv2.conf"`
```
Comment (blckbx) [inline FAQ.md:316]:
```suggestion
- Locate tunnelsatsv2.conf: `sudo find / -maxdepth 5 -type d \( -path /etc/wireguard -o -path /sd-root \) -prune -o -type f -name "tunnelsatsv2.conf" -print`
```

**Review by blckbx — COMMENTED** [review]
improving my own find command

**Review by blckbx — APPROVED** [review]
let's go 🚀
